packages/fxa-content-server/app/scripts/templates/sign_in_password.mustache

```diff
@@ -73,16 +73,25 @@
     {{/hasLinkedAccount}}
 
       <div id="tos-pp" class="text-grey-500 my-5 text-xs">
-        {{#isPocketClient}}
-          {{#unsafeTranslate}}
-            By proceeding, you agree to:<br />
-            Pocket’s <a class="link-grey" id="pocket-tos" href="https://getpocket.com/tos/">Terms of Service</a> and <a id="pocket-pp" class="link-grey" href="https://getpocket.com/privacy/">Privacy Notice</a><br />
-            Firefox’s <a class="link-grey" id="fxa-tos" href="/legal/terms">Terms of Service</a> and <a class="link-grey" id="fxa-pp" href="/legal/privacy">Privacy Notice</a>.
-          {{/unsafeTranslate}}
-        {{/isPocketClient}}
-        {{^isPocketClient}}
+        {{#defaultTermsOnly}}
           {{#unsafeTranslate}}By proceeding, you agree to the <a class="link-grey" id="fxa-tos" href="/legal/terms">Terms of Service</a> and <a class="link-grey" id="fxa-pp" href="/legal/privacy">Privacy Notice</a>.{{/unsafeTranslate}}
-        {{/isPocketClient}}
+        {{/defaultTermsOnly}}
+        {{^defaultTermsOnly}}
+          {{#isPocketClient}}
+            {{#unsafeTranslate}}
+              By proceeding, you agree to:<br />
+              Pocket’s <a class="link-grey" id="pocket-tos" href="https://getpocket.com/tos/">Terms of Service</a> and <a id="pocket-pp" class="link-grey" href="https://getpocket.com/privacy/">Privacy Notice</a><br />
+              Firefox’s <a class="link-grey" id="fxa-tos" href="/legal/terms">Terms of Service</a> and <a class="link-grey" id="fxa-pp" href="/legal/privacy">Privacy Notice</a>.
+            {{/unsafeTranslate}}
+          {{/isPocketClient}}
+          {{#isMonitorClient}}
+            {{#unsafeTranslate}}
+              By proceeding, you agree to:<br />
+              Firefox Monitor’s <a class="link-grey" id="monitor-tos" href="https://www.mozilla.org/privacy/firefox-monitor/">Terms of Service and Privacy Notice</a><br />
+              Firefox’s <a class="link-grey" id="fxa-tos" href="/legal/terms">Terms of Service</a> and <a class="link-grey" id="fxa-pp" href="/legal/privacy">Privacy Notice</a>
+            {{/unsafeTranslate}}
+          {{/isMonitorClient}}
+        {{/defaultTermsOnly}}
```

Comment:
I haven't checked this out locally yet, but I think I'm a little uncertain why we need `defaultTermsOnly`. Might be a weird case I'm not realizing but can we just use `isPocketClient` and `isMonitorClient` checks?

Probably not worth creating a template of sorts here though I imagine because Backbone.

Reply:
We could nest `^isPocketClient` and `^isMonitorClient` conditions, but this felt clearer to me, especially if we're moving towards adding more terms for relying parties. Loosely held opinion though!

Reply:
On further thought, I agree with you - `defaultTermsOnly` is not necessary. We can revise if/when we add terms for a third relying party. :)